css/style.css

```diff
@@ -76,6 +76,12 @@ body {
   list-style: none;
 }
 
+.content {
+  display: flex;
+  justify-content: center;
+  width: 50%;
+}
```

Comment:
- [ ] [REQUIRED] You have actually done a great job 👍🏽 but kindly note that the entire mobile version seems to have been disrupted with a wide margin on the right, or maybe you have fixed a wrong width. Please check. I guess this might have happed while you were trying to implement the desktop version.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/150560113-f88c94eb-715c-4a7d-99cf-cb9be5cf3326.png) | ![11](https://user-images.githubusercontent.com/58692590/150559884-816a1ceb-fd4b-4f52-a373-985cdce802a0.png)